main/Badge/index.tsx

```diff
@@ -17,7 +17,7 @@ export interface BadgeProps extends StyleProps {
   maximumCount?: number;
   showZero?: boolean;
   opacityVisible?: boolean;
-  variant?: string;
+  variant?: 'standard' | 'dot;
```

Comment:
You must miss the closing quotation mark(').

Reply:
@JeffGuKang